components/button.scss

```diff
@@ -1,20 +1,51 @@
+$button-font-size: 1rem;
 $button-radius: $border-radius;
 $button-border-width: 1px;
-$extra-border-hover: 1px;
 $button-padding-top-bottom: 10px;
 $button-padding-left-right: 16px;
 $button-padding: $button-padding-top-bottom $button-padding-left-right;
 $button-with-icon-font-size: 1.5rem;
 $icon-button-height: 36px;
 $icon-button-font-size: 22px;
 $icon-button-height-with-border: $icon-button-height - $button-border-width; // Subtract width of border to match size of buttons without border 
-$button-font-size: 1rem;
+$icon-button-large-height: 56px;
+$icon-button-notification-size: 18px;
+$icon-button-notification-radius: 20px;
+$extra-border-hover: 1px;
```

Comment:
Kan du separere (med en newline) grupper med variabler som hører sammen? Så for eks alle $icon-button blir sammen